lib/Borel/Error.hs

```diff
@@ -0,0 +1,54 @@
+{-# LANGUAGE OverloadedStrings #-}
+-- | Copyright 2013-2015 Anchor Systems, Pty Ltd and Others
```

Comment:
If you'd like to keep the header, can you update it to reflect borel's actual lifetime (2014-2015) and copy it to the other source files? (I'm fine with removing it too, up to you).